next/app/components/layouts/Header/Header.module.css

```
@@ -0,0 +1,23 @@
+.header {
+  display: flex;
+  background-color: #edc3e7;
+  font-size: 36px;
+  font-family: 'Montserrat';
+  font-weight: bold;
+  transform: scale(1.1, 1);
+  transform-origin: top left;
+  letter-spacing: -1.5px;
+  color: #2c3844;
+  padding: 20px;
```

実際に表示すると、少し大きめなので小さくしてほしいです！
```suggestion
  padding: 13px;
```